flutter/lib/src/default_integrations.dart

```diff
@@ -195,15 +196,95 @@ class _LoadContextsIntegrationEventProcessor extends EventProcessor {
         event = event.copyWith(contexts: eventContexts);
       }
 
-      if (infos['integrations'] != null) {
-        final integrations = List<String>.from(infos['integrations'] as List);
+      final tagsMap = infos['tags'];
```

Comment:
tags and extra set on Dart have precedence over the ones set in the native layer if the key exists on both sides